vscode-lean4/src/config.ts

```diff
@@ -201,6 +201,10 @@ export function getTestFolder() : string {
     return typeof(process.env.TEST_FOLDER) === 'string' ? process.env.TEST_FOLDER : '';
 }
 
+export function getTestVersion(): string | undefined {
```

Comment:
Should we just call this `DEFAULT_LEAN_TOOLCHAIN`? We do use it for tests but the implemented behaviour is to set `defaultToolchain` *in general*.

Reply:
Good point, I renamed it getDefaultLeanVersion and moved the default version here also.